pkg/timer_event/cleanup_users_marked_for_deletion.go

```diff
@@ -0,0 +1,96 @@
+package timer_event
+
+import (
+	"context"
+
+	"github.com/coneno/logger"
+	"github.com/influenzanet/go-utils/pkg/api_types"
+	"github.com/influenzanet/go-utils/pkg/constants"
+	loggingAPI "github.com/influenzanet/logging-service/pkg/api"
+	messageAPI "github.com/influenzanet/messaging-service/pkg/api/messaging_service"
+	"github.com/influenzanet/user-management-service/pkg/utils"
+)
+
+// CleanupUsersMarkedForDeletion handles the deletion of accounts that did not react to reminder mail
+func (s *UserManagementTimerService) CleanupUsersMarkedForDeletion() {
+	logger.Debug.Println("Starting clean up job for users marked for deletion:")
+	instances, err := s.globalDBService.GetAllInstances()
+	if err != nil {
+		logger.Error.Printf("unexpected error: %s", err.Error())
+	}
+	for _, instance := range instances {
+		users, err := s.userDBService.FindUsersMarkedForDeletion(instance.InstanceID)
+		count := 0
+
+		if err != nil {
+			logger.Error.Printf("unexpected error: %s", err.Error())
+			continue
+		}
+		for _, u := range users {
+			// ---> Trigger message sending
+			_, err = s.clients.MessagingService.SendInstantEmail(context.TODO(), &messageAPI.SendEmailReq{
+				InstanceId:        instance.InstanceID,
+				To:                []string{u.Account.AccountID},
+				MessageType:       constants.EMAIL_TYPE_ACCOUNT_DELETED_AFTER_INACTIVITY,
```

Comment:
Could you please add the new email templates into the changelog, so we have an overview of all the things that are necessary to get this running in one place?

Reply:
Email templates are added. Can you please check if CHANGELOG.md text is sufficient or if some information is still missing?